release-1.6/release-notes-draft.md

```diff
@@ -90,19 +93,25 @@
   ([proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/bootstrap-discovery.md))
   which is baked into the Kubernetes API server, rather than a separate
   service.
-- kubeadm now supports managing the bootstrap tokens active in your cluster
-  using the new [`kubeadm token`](https://kubernetes.io/docs/admin/bootstrap-tokens/) subcommands.
 - Many other bug-fixes and improvements have been contributed that make using
   `kubeadm` a more pleasant and reliable experience.
+- The config file (still in alpha) now supports passing arbitrary flags to the
+  control plane components.
 
 ### Breaking Changes
+- Quite a few flags been renamed or removed.  Those options that are removed as
+  flags can still be accessed via the config file.  Most noteably this includes
+  external etcd settings and the option for setting the cloud provider on the
+  API server.  The [kubeadm reference
+  documentation](https://kubernetes.io/docs/admin/kubeadm/) is up to date with
+  the new flags.
 - A new label and taint is used for marking the master. The label is
   `node-role.kubernetes.io/master=""` and the taint has the effect
```

Comment:
We should remove `and the taint has the effect NoSchedule`, it doesn't make sense in this sentence.